### PR TITLE
Fhac 297

### PR DESCRIPTION
**Pull request by jsmith2**
Fix for reading internal connection info for FHIR resources.  If there are no resources, configuring left nav menu to not have link and to block all connections to the FHIR page.

Also, added updated progress gif (replaces demo fhir version).

### CODE REVIEW

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/display/FhirDisplayController.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    public void checkDisplay() {
+        String[] resourceNames = new String[]{FhirResourceService.BINARY_RESOURCE_NAME, FhirResourceService.DOCREF_RESOURCE_NAME,
+            FhirResourceService.PATIENT_RESOURCE_NAME };
+        
+        for(String resourceName : resourceNames) {
+            boolean hasResource;
+            try {
+                hasResource = checkForResource(resourceName);
+            } catch(ConnectionManagerException e) {
+                LOG.warn(e, e);
```
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/display/FhirDisplayController.java]: Should this be "LOG.warn(e.getLocalizedMessage(), e)"?

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/display/FhirDisplayController.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                hasResource = false;
+            }
+            
+            if(hasResource == true) {
```
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/display/FhirDisplayController.java]: "== true" is redundant here.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/display/FhirDisplayController.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+public class FhirDisplayController implements DisplayController {
+    
+    private static final Logger LOG = Logger.getLogger(FhirDisplayController.class);
+
+    @Override
+    public void checkDisplay() {
+        String[] resourceNames = new String[]{FhirResourceService.BINARY_RESOURCE_NAME, FhirResourceService.DOCREF_RESOURCE_NAME,
+            FhirResourceService.PATIENT_RESOURCE_NAME };
+        
+        for(String resourceName : resourceNames) {
```
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/display/FhirDisplayController.java]: Do you have the NetBeans formatter file imported?  Lots of small things with spacing across multiple files.

**Comment by christophermay07** [comment]
I've reviewed this PR, endorsing for merge without further comment.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/display/FhirDisplayController.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import gov.hhs.fha.nhinc.connectmgr.ConnectionManagerException;
+import gov.hhs.fha.nhinc.nhinclib.NhincConstants;
+import gov.hhs.fha.nhinc.nhinclib.NullChecker;
+import org.apache.log4j.Logger;
+
+/**
+ *
+ * @author jassmit
+ */
+public class FhirDisplayController implements DisplayController {
```
Comment (nareshcgi) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/display/FhirDisplayController.java:40]: We should have a better way to do this going forward, make it in a single place or rethink this approach. Also we should change the name from Controller, it means something different in JSF.

**Comment by nareshcgi** [comment]
Reviewed the PR and looks good.

**Pull request merged**